cli/sync_service/object.go

```diff
@@ -5,104 +5,170 @@ import (
 	"fmt"
 	"github.com/open-horizon/anax/cli/cliutils"
 	"github.com/open-horizon/anax/config"
-	"github.com/open-horizon/anax/exchange"
 	"github.com/open-horizon/anax/i18n"
 	"github.com/open-horizon/edge-sync-service/common"
 	"net/http"
 	"os"
 	"path"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
 )
 
 type MMSObjectInfo struct {
-	Definition   common.MetaData             `json:"definition"`
+	ObjectID     string                      `json:"objectID,omitempty"`
+	ObjectType   string                      `json:"objectType,omitempty"`
+	Definition   *common.MetaData            `json:"definition,omitempty"`
 	Destinations []common.DestinationsStatus `json:"destinations,omitempty"`
+	ObjectStatus string                      `json:"objectStatus,omitempty"`
 }
 
-// Display the object metadata for a given object in the MMS.
-func ObjectList(org string, userPw string, objType string, objId string, details bool) {
+// Display the object metadata for given flags in the MMS.
```

Comment:
In e2edev, the command "hzn mms object list -d -l" returns the same object 5 times. Specifying just -d or just -l, or neither does not expose the bug.

Reply:
resolved in latest push